pycon/settings.py

```diff
@@ -4,16 +4,18 @@
 import os.path
 import sys
 
+from decouple import config
+from unipath import Path
```

Comment:
if we're adding some path management libraries, I'd got with pathlib instead of unipath, especially that its included in python3 (and that it has overloaded '/' which looks better than .child()).

that said – I'd put that in a separate PR anyway and keep this one just about `config()` vs `os.environ`

Reply:
Both good points, will undo the path changes.